modules/civiremote_event/civiremote_event.routing.yml

```diff
@@ -123,3 +123,15 @@ civiremote_event.checkin_form:
     parameters:
       checkin_token:
         type: civiremote_checkin_token
+
+civiremote_event.mailing_list_confirm_subscription:
+  path: '/civiremote/event/mailing/confirm/{token}'
```

Comment:
Let's be a little more specific and use `mailing-list-subscribe` or the like instead of just `mailing`.

Reply:
Changed.